src/python-common/ceph/deployment/translate.py

```diff
@@ -15,11 +15,13 @@ class to_ceph_volume(object):
 
     def __init__(self,
                  spec,  # type: DriveGroupSpec
-                 selection  # type: DriveSelection
+                 selection,  # type: DriveSelection
+                 host  # type: str
```

Comment:
Adding a `host` parameter to `to_ceph_volume()` feels a bit wrong, as `to_ceph_volume()` is supposed to be independent of the host. What about directly passing `reclaimed_ids` as a parameter and then:

```python
to_ceph_volume(
    spec,
    selection,
    self.spec.osd_id_claims.get(self.host, [])
)
```
?

Reply:
agreed